src/components/DetailSidebar/DetailSidebarFundButton.tsx

```diff
@@ -21,21 +22,21 @@ const StyledFundStrong = styled('strong')({
   marginRight: 3,
 });
 
-interface Props {
-  to: string;
-}
-
 /* eslint-disable react/jsx-no-bind */
-const DetailSidebarFundButton: React.FC<Props> = ({ to }) => {
-  const preventDefault = (event: MouseEvent<HTMLButtonElement>) => event.preventDefault();
+const DetailSidebarFundButton: React.FC = () => {
+  const detailContext = useContext(DetailContext);
+
+  const { packageMeta } = detailContext;
+
+  if (!packageMeta?.latest?.funding) {
```

Comment:
Maybe add the `.url` check too?

Reply:
According npmjs https://github.com/npm/rfcs/pull/54/files#diff-1bc199539d172d3211c84688a617e18cR30 should be there ... can we trust on them?

Reply:
> supports a string or an object

We have to test if the metadata always contains / uses `url`.

Reply:
I will